spec/std/bit_array_spec.cr

```diff
@@ -8,6 +8,13 @@ private def from_int(size : Int32, int : Int)
   ba
 end
 
+private def assert_no_unused_bits(ba : BitArray, *, file = __FILE__, line = __LINE__)
+  bit_count = (ba.size - 1) // 32 + 1
+  (ba.size...bit_count).each do |index|
```

Comment:
Perhaps I'm just missing something here, but it looks like ˋbit_countˋ would never be bigger than ˋba.sizeˋ (and ˋRange#eachˋ doesn't go backwards, although that's probably not intended anyways).

Reply:
In other words: are we shure this assertion would actually detect unused bits? ;)